presto-docs/src/main/sphinx/release/release-0.122.rst

```diff
@@ -0,0 +1,12 @@
+=============
+Release 0.122
+=============
+
+General Changes
+---------------
+
+* In release 0.117, casts between JSON and VARCHAR were added back to provide
```

Comment:
I would shorten this to say: "The deprecated casts between JSON and VARCHAR will now fail and provide the user with instructions to migrate their query. For more details, see :doc:`/release/release-0.116`."